src/platform.ts

```diff
@@ -66,21 +66,49 @@ export function getDefaultPowerShellPath(
     use32Bit: boolean = false): string | null {
 
     let powerShellExePath;
+    let psCoreInstallPath;
 
     // Find the path to powershell.exe based on the current platform
```

Comment:
This comment needs updating now
```suggestion
     // Find the path to the powershell executable based on the current platform
```